grammar.js

```diff
@@ -644,23 +645,64 @@ module.exports = grammar({
 
     module_body: $ => seq(
       '{',
-      repeat($.module_directive),
+      repeat(
+        seq($.module_directive, ';')
```

Comment:
Could you remove the `;` from here, and instead add it at the end of each type of module directive? It is helpful to have the `;` node grouped with the directive, similar to other declarations and statements.

Reply:
I agree. Thanks for the catch.